trie/verkle.go

```diff
@@ -239,7 +237,7 @@ func nodeToDBKey(n verkle.VerkleNode) []byte {
 // Commit writes all nodes to the trie's memory database, tracking the internal
 // and external (for account tries) references.
 func (trie *VerkleTrie) Commit(onleaf LeafCallback) (common.Hash, int, error) {
-	flush := make(chan verkle.VerkleNode)
+	flush := make(chan verkle.VerkleNode, 64)
```

Comment:
Deep into the pprof traces, I noticed some contention in this channel which makes sense if you think about it. It wasn't meaningless nor double-digit %, but does a reasonable improvement.